ml-fundamentals/ml-feasibility-study.md

```diff
@@ -0,0 +1,98 @@
+# ML Feasibility Studies
+
+The main goal of Machine Learning (ML) feasibility studies is to assess whether the problem in question is feasible to solve satisfactorily using ML with the available data. They work to mitigate the potential for over-investment before there exists:
+
+* Sufficient evidence that a ML solution would be the best technical solution given the business case
+* Sufficient evidence that a ML solution is possible
+* Some vetted direction on what a ML solution should look like
+
+This effort ensures quality solutions backed by the appropriate, thorough amount of consideration and evidence.
+
+## When are ML feasibility studies useful?
+
+Every engagement with a ML component, potentially excluding purely ML Ops engagements focusing on operationalization of existing solutions, can benefit from a ML feasibility study early in the project. Architectural discussions can still occur in parallel as the team works towards a gaining solid understanding and definition of what will be built. Feasibility studies can last between 3-12 weeks, depending on specific problem details, volume of data, state of the data etc. Starting with a 3 week milestone might be useful, during which it can be determined how much more time, if any, is required for completion.
```

Comment:
```suggestion
Every engagement with a ML component, potentially excluding pure ML Ops engagements, can benefit from a ML feasibility study early in the project.

Architectural discussions can still occur in parallel as the team works towards a gaining solid understanding and definition of what will be built.

Feasibility studies can last between 3-12 weeks, depending on specific problem details, volume of data, state of the data etc. Starting with a 3 week milestone might be useful, during which it can be determined how much more time, if any, is required for completion.
```